curvefs/src/mds/chunkid_allocator.cpp

```diff
@@ -26,20 +26,24 @@
 namespace curvefs {
 namespace mds {
 
-int ChunkIdAllocatorImpl::GenChunkId(uint64_t *chunkId) {
+int ChunkIdAllocatorImpl::GenChunkId(uint64_t idNum, uint64_t *chunkId) {
     int ret = 0;
     // will unloack at destructor
     ::curve::common::WriteLockGuard guard(nextIdRWlock_);
 
-    if (nextId_ > lastId_ || CHUNKIDINITIALIZE == nextId_) {
+    if (nextId_ + idNum > lastId_ || CHUNKIDINITIALIZE == nextId_) {
         // the chunkIds is exhausted || first to get chunkId
-        ret = AllocateBundleIds(bundleSize_);
+        uint64_t allocSize = nextId_ + idNum - lastId_;
```

Comment:
allocSize = ((nextId_ + idNum - lastId_)/bundleSize_ +1)*bundleSize_ ?

Otherwise, the next application will still cause resource exhaustion to trigger allocation